SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialConnectionManager.cpp

```diff
@@ -412,7 +413,11 @@ void USpatialConnectionManager::ConnectToReceptionist(uint32 PlayInEditorID)
 
 	ConfigureConnection ConnectionConfig(ReceptionistConfig, bConnectAsClient);
 
-	TSharedPtr<SpatialGDK::SpatialEventTracer> EventTracer = CreateEventTracer(ReceptionistConfig.WorkerId);
+	TSharedPtr<SpatialGDK::SpatialEventTracer> EventTracer = nullptr;
+	if (USpatialGameInstance* GameInstance = GetTypedOuter<USpatialGameInstance>())
+	{
+		EventTracer = GameInstance->CreateEventTracer(ReceptionistConfig.WorkerId);
```

Comment:
Wonder if we should create the event tracer from the same point in code for native and spatial (game instance) and just get it here? Is there a reason we need to create it at this point?